core/src/main/resources/org/verapdf/report/DetailedHtmlReport.xsl

```diff
@@ -169,8 +169,14 @@
                         <b>Version:</b>
                     </td>
                     <td>
-                        <xsl:value-of
-                                select="/report/buildInformation/releaseDetails[@id='gui']/@version"/>
+                        <xsl:choose>
```

Comment:
It would be more logical to pass the required @id value as a parameter to the XSLT transform